lib/tasks/import.rake

```diff
@@ -0,0 +1,91 @@
+require 'csv'
+
+namespace :csv_load do
+  task customers: :environment do
+    CSV.foreach('./db/data/customers.csv', headers: true) do |row|
+      Customer.create!(row.to_hash)
+    end
+  end
+
+  task invoice_items: :environment do
+    CSV.foreach('./db/data/invoice_items.csv', headers: true) do |row|
```

Comment:
So the |row| in this case refers to the columns?

Reply:
No, the row will be a single row containing one cell from each column. Those are combined to make an object. Then we move to the next row and make a new object from the next row. The columns will be all one type of data i.e. first_names.

Reply:
kk, that makes sense! Thanks!